.github/workflows/release.yml

```
@@ -29,6 +29,16 @@ jobs:
           go-version: 1.19.x
           check-latest: true
           cache: true
+        # uses https://cloud.google.com/iam/docs/workload-identity-federation to
+        # swap a Github OIDC token for GCP service account credentials, allowing
+        # this workflow to manage the deploy bucket
+      - name: Auth To GCP
+        uses: google-github-actions/auth@v0
+        with:
+          workload_identity_proider: projects/491113660045/locations/global/workloadIdentityPools/plugins-workload-pool/providers/plugins-workload-provider
+          service_account: buf-plugins-1-publisher@buf-plugins-1.iam.gserviceaccount.com
+      - name: Setup GCLOUD
+        uses: 'google-github-actions/setup-gcloud@v0'
```

```suggestion
        uses: 'google-github-actions/setup-gcloud@v1'
```

v1 appears to be the latest release - however we're pinning sha hashes for all 3rd party actions in this project.